persistence-dse-6.8/src/main/java/io/stargate/db/dse/DsePersistenceActivator.java

```diff
@@ -125,6 +125,19 @@ public static Config makeConfig(File baseDir) throws IOException {
       c.guardrails = guardrailsConfig;
     }
 
+    long timeout;
+    if ((timeout = Long.getLong("stargate.request_timeout_in_ms", -1)) > 0) {
+      c.request_timeout_in_ms = timeout;
+    }
+
+    if ((timeout = Long.getLong("stargate.write_request_timeout_in_ms", -1)) > 0) {
+      c.write_request_timeout_in_ms = timeout;
+    }
+
+    if ((timeout = Long.getLong("stargate.read_request_timeout_in_ms", -1)) > 0) {
```

Comment:
do we want the default to be `-1`? Will it mean that it waits infinitely?

Reply:
this is just for the `if` it will not be used. I'm also going to refactor this to use config files when #602 merges.

Reply:
I'll follow up with a more generic solution in another PR.